SIRF_data_preparation/data_QC.py

```diff
@@ -73,10 +73,10 @@ def plot_image(image, save_name=None, transverse_slice=-1, coronal_slice=-1, sag
         coronal_slice = image.dimensions()[1] // 2
     if sagittal_slice < 0:
         sagittal_slice = image.dimensions()[2] // 2
+    arr = image.as_array()
     if vmax is None:
-        vmax = image.max()
+        vmax = np.percentile(arr, 99) / .99
```

Comment:
wait why `/.99`?

```suggestion
        vmax = np.percentile(arr, 99)
```

or maybe you meant:

```suggestion
        vmax = np.percentile(arr, 99.995)
```

Reply:
I divided by .99 just to get it a bit closer to the original max (only works if linear distribution of course).